File: 2/Product.cpp
#include <iostream>
#include <string>

using namespace std;

class Product
{
	private:
		int price;
		string idProduct;

	public:
		// construct
		Product(int price, string idProduct)
		{
			this->price = price;
			this->idProduct = idProduct;
		}

		// default construct
		Product(){}

		void setPrice(int price)
		{
			this->price = price;
		}

		int getPrice()
		{
			return this->price;
		}

		void setIdProduct(string idProduct)
		{
			this->idProduct = idProduct;
		}

		string getIdProduct()
		{
			return this->idProduct;
		}

		// deconstruct
		~Product(){}

		void printProduct()
		{
			cout << "Harga : " << this->price << endl;
			cout << "ID Produk : " << this->idProduct << endl;
		}
};

File: 1/Product.java
class Product
{
	private int price;
	private String idProduct;
	
	public Product(int price, String idProduct)
	{
		this.setPrice(price);
		this.setIdProduct(idProduct);
	}

	public Product(){}

	public String getIdProduct()
	{
		return idProduct;
	}

	public void setIdProduct(String idProduct)
	{
		this.idProduct = idProduct;
	}

	public int getPrice()
	{
		return price;
	}

	public void setPrice(int price)
	{
		this.price = price;
	}

	public void printProduct()
	{
		System.out.println("Harga : " + this.price);
		System.out.println("ID Produk : " + this.idProduct);
	}
}